src/documentation/pages/components/forms/search-input/examples/_sizes-lumen-centurylink.vue

```diff
@@ -0,0 +1,158 @@
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -xs" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>
+<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -sm" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -sm -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -sm" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>
+<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -md" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -md -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -sm" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>
+<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -lg" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -lg -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -sm" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>
+<div class="chi-form__item">
+  <div class="chi-input__wrapper -icon--right">
+    <input class="chi-input chi-search__input -xl" type="search" value="Sample text" aria-label="search input" />
+    <button class="chi-button -icon -flat -xl -bg--none" aria-label="Search">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-search" aria-hidden="true"></i>
+      </div>
+    </button>
+    <button class="chi-button -icon -close -sm" aria-label="Close">
+      <div class="chi-button__content">
+        <i class="chi-icon icon-x" aria-hidden="true"></i>
+      </div>
+    </button>
+  </div>
+</div>`,
+      },
+    };
+  },
+})
+export default class SizesLumenCenturyLink extends Vue {}
```

Comment:
looks like the web component code snippet and the vue code snippet are the same, when they should be different